builds/misc/images-release.yaml

```diff
@@ -526,11 +515,11 @@ jobs:
               ]
           signConfigType: inlineSignParams
       - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
-        displayName: "azureiotedge-diagnostics Code Sign"
-        inputs: 
+        displayName: "dotnet-azureiotedge-diagnostics Code Sign"
```

Comment:
Why do you change the step name?

Reply:
I guess I didn't have to change the name, I was just going through and changing everything and got the name as well